web/analytics/export.py

```
@@ -0,0 +1,615 @@
+import csv
+import io
+import json
+from datetime import datetime
+from django.http import HttpResponse
+from reportlab.lib.pagesizes import letter
+from reportlab.lib import colors
+from reportlab.lib.styles import getSampleStyleSheet, ParagraphStyle
```

Ruff (0.8.2)

`reportlab.lib.styles.ParagraphStyle` imported but unused

Remove unused import: `reportlab.lib.styles.ParagraphStyle`

(F401)